pyrinter/document.py

```diff
@@ -225,7 +227,9 @@ def __get_text_size(text: str, font: Font) -> float:
         :return: the width of the text
         """
         root = Tk()  # Needed to estimate the width.
-        font_var = TkFont(family=font.font_name, size=font.height, weight="normal")
+        if font.weight != "bold": # Make sure we have a valid weight be defaulting to "normal"
+            font.weight = "normal"
+        font_var = TkFont(family=font.font_name, size=font.height, weight=font.weight)
```

Comment:
If statement used for backwards compatibility, if nothing or invalid text is entered, normal is defaulted.